modules/passkey/test/userstories/OffchainPasskeyVerification.spec.ts

```diff
@@ -0,0 +1,105 @@
+import { expect } from 'chai'
+import { deployments, ethers } from 'hardhat'
+import { WebAuthnCredentials, decodePublicKey, encodeWebAuthnSignature } from '../utils/webauthn'
+import { buildSignatureBytes } from '@safe-global/safe-4337/src/utils/execution'
+
+describe('Offchain Passkey Signature Verification [@userstory]', () => {
```

Comment:
One thing that @akshay-ap did that I thought was really nice in the other user story PRs was to have a documentation header on the `describe` block that explains the user story. For example:

https://github.com/safe-global/safe-modules/blob/810e0732020fca02e2fd6310e7e354d4dbb700eb/modules/passkey/test/userStories/createSafeWithPasskeySignerAsOwner.ts#L9-L17

Reply:
Done 👍🏾